areas/deploy/src/AzureMcp.Deploy/DeploySetup.cs

```diff
@@ -22,21 +24,36 @@ public void ConfigureServices(IServiceCollection services)
+            + "- iac_rules_get: offers guidelines for creating Bicep/Terraform files to deploy applications on Azure; "
+            + "- app_logs_get: fetch logs from log analytics workspace for Container Apps, App Services, function apps that were deployed through azd; "
+            + "- pipeline_guidance_get: guidance to create a CI/CD pipeline which provision Azure resources and build and deploy applications to Azure; "
+            + "- architecture_diagram_generate: generates an azure service architecture diagram for the application based on the provided app topology; ");
         rootGroup.AddSubGroup(deploy);
 
-        deploy.AddCommand("plan-get", new PlanGetCommand(loggerFactory.CreateLogger<PlanGetCommand>()));
-
-        deploy.AddCommand("iac-rules-get", new IaCRulesGetCommand(loggerFactory.CreateLogger<IaCRulesGetCommand>()));
-
-        deploy.AddCommand("azd-app-log-get", new AzdAppLogGetCommand(loggerFactory.CreateLogger<AzdAppLogGetCommand>()));
-
-        deploy.AddCommand("cicd-pipeline-guidance-get", new PipelineGenerateCommand(loggerFactory.CreateLogger<PipelineGenerateCommand>()));
-
-        deploy.AddCommand("architecture-diagram-generate", new GenerateArchitectureDiagramCommand(loggerFactory.CreateLogger<GenerateArchitectureDiagramCommand>()));
+        // Application-specific commands
+        var appGroup = new CommandGroup("app", "Application-specific deployment tools");
+        appGroup.AddCommand("logs_get", new LogsGetCommand(loggerFactory.CreateLogger<LogsGetCommand>()));
+        deploy.AddSubGroup(appGroup);
+
+        // Infrastructure as Code commands
+        var infrastructureGroup = new CommandGroup("iac", "Infrastructure as Code operations");
+        infrastructureGroup.AddCommand("rules_get", new RulesGetCommand(loggerFactory.CreateLogger<RulesGetCommand>()));
+        deploy.AddSubGroup(infrastructureGroup);
+
+        // CI/CD Pipeline commands
+        var pipelineGroup = new CommandGroup("pipeline", "CI/CD pipeline operations");
+        pipelineGroup.AddCommand("guidance_get", new GuidanceGetCommand(loggerFactory.CreateLogger<GuidanceGetCommand>()));
+        deploy.AddSubGroup(pipelineGroup);
+
+        // Deployment planning commands
+        var planGroup = new CommandGroup("plan", "Deployment planning operations");
+        planGroup.AddCommand("get", new GetCommand(loggerFactory.CreateLogger<GetCommand>()));
+        deploy.AddSubGroup(planGroup);
+
+        // Architecture diagram commands
+        var architectureGroup = new CommandGroup("architecture", "Architecture diagram operations");
+        architectureGroup.AddCommand("diagram_generate", new DiagramGenerateCommand(loggerFactory.CreateLogger<DiagramGenerateCommand>()));
+        deploy.AddSubGroup(architectureGroup);
```

Comment:
Not sure if this is done, but please review my guidance on naming these.  We won't have underscores in the names.  They will be nested.

Reply:
It should have a "get"